src/index.ts

```diff
@@ -1,2 +1,8 @@
-export * from './vanilla'
-export * from './react'
+export { useSnapshot } from "./react"
```

Comment:
```suggestion
// export * from './vanilla'
// export * from './react'
// NOTE snowpack doesn't allow export *, so we name all instead
export { useSnapshot } from "./react"
```

Reply:
I already added this, I think

Reply:
@barelyhuman I change my mind. We are fine with named re-exports, which is more explicit. Let me remove the comments.

Reply:
Cool 👍🏻